tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+    "compilerOptions": {
+      "module": "commonjs",
+      "esModuleInterop": true,
```

Comment:
Any particular reason you need this enabled?

Reply:
Ah is it so that you can use default imports for js packages? i.e. `import env from 'dotenv'` instead of `import * as env from 'dotenv'` ?

Reply:
not actually sure haha. I copied that config from another example project

Reply:
Better answer. I get this error on imports when I remove that line:

> index.d.ts(1093, 1): This module is declared with using 'export =', and can only be used with a default import when using the 'esModuleInterop' flag.

Reply:
So yeah what you said is probably true